Android/src/org/droidplanner/android/fragments/mode/ModeFollowFragment.java

```diff
@@ -232,6 +232,8 @@ public void onScrollingEnded(CardWheelHorizontalView cardWheel, LengthUnit oldVa
     public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
         final FollowType type = adapter.getItem(position);
 
+        DroidPlannerPrefs.getInstance(null).setLastKnownFollowType(type);
```

Comment:
Same comment here as above. You can use `getAppPrefs()` defined by its parent instead.